scripts/apps/authoring-react/article-widgets/translations/translations.tsx

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
+import {RelativeDate} from 'core/datetime/relativeDate';
+import {state as State} from 'apps/search/components/fields/state';
+import {IArticle, IArticleSideWidget, IExtensionActivationResult, IRestApiResponse} from 'superdesk-api';
+import {gettext} from 'core/utils';
+import {httpRequestJsonLocal} from 'core/helpers/network';
+import {openArticle} from 'core/get-superdesk-api-implementation';
+import {AuthoringWidgetLayout} from 'apps/dashboard/widget-layout';
+import {AuthoringWidgetHeading} from 'apps/dashboard/widget-heading';
+import {Card} from 'core/ui/components/Card';
+import {Spacer, SpacerBlock} from 'core/ui/components/Spacer';
+import {Label} from 'superdesk-ui-framework';
+
+const getLabel = () => gettext('Translations');
+
+type IProps = React.ComponentProps<
+    IExtensionActivationResult['contributions']['authoringSideWidgets'][0]['component']
+>;
+
+interface IState {
+    translations: Array<IArticle>;
+    translationsLookup: Dictionary<IArticle['_id'], IArticle>;
+}
+
+class Translations extends React.Component<IProps, IState> {
```

Comment:
Looks like you mostly copy-pasted code from `TranslationsWidgetComponent`. Would it work to reusing that component here instead? Try doing this. It looks to me like HTML markup is different, but it may work to pass `TranslationItemTemplate` and `TranslationsWrapperTemplate` and reuse the rest.